app/pocket/main.go

```diff
@@ -22,6 +23,12 @@ func main() {
 	}
 
 	cfg := config.LoadConfig(*config_filename)
+	// DISCUSS(deblasis): poor man's feature flag: no port => no RPC
```

Comment:
Maybe adding an `EnableRPC` in the `RPCConfig` we'll add?

Reply:
Done